app/apollo/utils/bunyan.js

```diff
@@ -40,9 +40,9 @@ const getBunyanConfig = (route) => {
   const result = {
     name: route,
     parseUA: false,
-    excludes: ['referer', 'body', 'short-body'],
+    excludes: ['referer', 'short-body'],
     levelFn: responseCodeMapper,
-    obfuscate: ['req.headers.razee-org-key'],
+    obfuscate: ['req.headers.razee-org-key', 'req.body.variables'],
```

Comment:
We still need to see variables because that helps us understand more about a query especially when someone passes in the custom `mongoQuery` string as a variable.